dev/package.json

```diff
@@ -1,8 +1,11 @@
 {
-  "name": "Flapjack",
+  "name": "syrup-test",
   "version": "1.0.0",
   "description": "A simple test application for syrup.",
   "main": "gulpfile.js",
+  "homepage": "http://allenai.github.io/syrup/",
+  "repository": "allenai/syrup",
+  "license": "GPL-2.0",
```

Comment:
This might require us open-source Aristo. I'm not sure the implications of using a javascript library (which by definition is source) that's GPL.